docs/introduction.md

```diff
@@ -0,0 +1,232 @@
+# Introduction
+
+This document is aimed at introducing developers to the Flow Data Provisioning Service project.
+
+## Table of Contents
+
+1. [Getting Started](#getting-started)
+3. [What is Flow](#flow)
+    1. [Glossary](#glossary)
+4. [Developer Guide](#developer-guide)
+    1. [Installation](#installation)
+    2. [Setting up a test environment](#setting-up-a-test-environment)
+5. [More resources](#more-resources)
+
+## Getting Started
+
+The Flow Data Provisioning Service (DPS) is a web service that maintains and provides access to the history of the Flow [execution state](#execution-nodes).
+
+The reason for this need is that in the execution state, blocks are pruned after a hundred other blocks have been finalized, which makes it impossible to visualize the history of the chain.
```

Comment:
"the in-memory execution state is pruned after 300 chunks".
"which makes it impossible to access the state history".